src/Components/FavoritesList/FavoritesList.js

```diff
@@ -0,0 +1,31 @@
+import { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { checkStorage, deleteFromStorage} from '../../localStorageHandlers'
+import './FavoritesList.css';
+
+const FavoritesList = (props) => {
+  const [monsters, setMonsters] = useState([])
+
+  useEffect(() => {
```

Comment:
FavoritesList looks very clean - I like your use of useEffect here to pull in stored favorites.